package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -0,0 +1,293 @@
+"""
+Secondary structure assignment (helix, sheet
+and loop) --- :mod:`MDAnalysis.analysis.dssp`
+==========================================================================
```

Comment:
Does this format properly or does it have to be on a single line?

Reply:
oh, I thought docs are invincible for single line breaks, since they have to follow the max line length requirement🤔
anyway, made it a single line